packages/contentstack-import/src/import/modules-js/global-fields.js

```diff
@@ -114,9 +114,9 @@ module.exports = class ImportGlobalFields {
         })
         .catch(function (err) {
           // error while importing globalfields
-          let error = JSON.parse(err);
+          let error = err;
```

Comment:
Are we using error variable here, if not, then we can remove this initialization

Reply:
Removed it @shafeeqd959